### DISCUSSION TITLE
ServiceStation: Safari Edition?

### DISCUSSION THREAD

**Discussion by JDiaz11** [knurling/ServiceStation#64] [Q&A]
Hi Buddy-- I'm really impressed by what you've done here! I'm looking to accomplish something very similar in Safari's contextual link menu. Is there any way you think I'd be able to reverse engineer the Finder ServiceStation to accomplish this? I'm not too experienced in XCode. I'm pretty proficient with AppleScripting, somewhat with Shell scripting and pretty good with VB and PowerShell. So I'm comfortable tweaking and poking around, but as the Developer you obviously have a better sense of whether what I'm attempting is even possible. Thanks so much! J

**Comment by gingerbeardman** [comment]
> something very similar in Safari's contextual link menu

Extensions can add items to Safari context menu.

But what has that to do with ServiceStation?

What exactly are you wanting to do from the Safari context menu?
